tools/clang/tools/dxcompiler/dxcompilerobj.cpp

```diff
@@ -161,7 +161,7 @@ struct CompilerVersionPartWriter {
 };
 
 static HRESULT CreateContainerForPDB(IMalloc *pMalloc,
-  llvm::Module *pModule,
+  std::unique_ptr<llvm::Module> pModule,
```

Comment:
Why is this function taking ownership of the module now?  The module is only used here to get the shader model, so it can construct a program part for the reflection stream.  However, that is totally unnecessary, since we already constructed the reflection blob earlier under AssembleToContainer.  We should just retain that and use it, rather than regenerate it again for the part in the slim PDB.